src/impl/collection_change_builder.cpp

```diff
@@ -282,7 +282,7 @@ void CollectionChangeBuilder::verify()
 
 namespace {
 struct RowInfo {
-    int64_t obj_key;
+    int64_t key;
```

Comment:
This change from obj_key to key is because it's sometimes an object key, and sometimes an index in a list.